### PR TITLE
New Adapter: Oftmedia (alias of adtelligent)

### PR DESCRIPTION
**Pull request by GeneGenie**
Adding new adapter alias

Doc: https://github.com/prebid/prebid-server/pull/2302

### CODE REVIEW

File: static/bidder-info/oftmedia.yaml
Comment (SyntaxNode) [inline static/bidder-info/oftmedia.yaml:2]: Please use a group email address rather than an individual.

**Comment by AlexBVolcy** [comment]
Looks good to me, just waiting on e-mail address update before approval

File: static/bidder-info/oftmedia.yaml
Comment (GeneGenie) [reply static/bidder-info/oftmedia.yaml]: done

**Comment by SyntaxNode** [comment]
@GeneGenie Thank you for changing the email. Last request: Could you please update the [adapter docs ](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/oftmedia.md) to indicate the PBS adapter is an alias of Adtelligent?

Looks good otherwise.

**Comment by GeneGenie** [comment]
> @GeneGenie Thank you for changing the email. Last request: Could you please update the [adapter docs ](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/oftmedia.md) to indicate the PBS adapter is an alias of Adtelligent?
> 
> Looks good otherwise.

The only question - how to make this indication. Coz client module, aliasing another bidAdapter code.

**Comment by AlexBVolcy** [comment]
@GeneGenie I think you would just add an `aliasCode: alias_name` section to your adapter docs, but I think you've already done that correctly. @SyntaxNode can you confirm? 

I looked at another adapter doc that is an alias, and they followed that same format. https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/janet.md

**Comment by SyntaxNode** [comment]
This was asked in Prebid Slack and the answer given is:
> You'll need to have two versions: oftmedia.md and oftmedia_server.md.

**Comment by GeneGenie** [comment]
Added doc

**Review by SyntaxNode — APPROVED** [review]

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**
